src/codegen/mod.rs

```diff
@@ -3127,6 +3127,13 @@ impl CodeGenerator for Function {
         debug!("<Function as CodeGenerator>::codegen: item = {:?}", item);
         debug_assert!(item.is_enabled_for_codegen(ctx));
 
+        // We can't currently do anything with Internal functions so just
+        // them.
```

Comment:
So just avoid generating them?

Also I wonder if it looks nicer to just do `self.linkage() != Linkage::Internal`.